docs/faq.md

```diff
@@ -4,6 +4,10 @@ sidebar_position: 2
 
 # Frequently asked questions
 
+## What is conda?
+
+[Conda](https://docs.conda.io/en/latest/) is an OS-agnostic, [open source](https://github.com/conda/conda#readme) package manager and environment management system that helps you find and install packages as well as their dependencies. Though conda is written in Python and was created mainly for Python programs, it can package and distribute software for any language.
```

Comment:
```suggestion
[Conda](https://docs.conda.io/) is an [open-source](https://github.com/conda/conda#readme) package manager and environment management system that helps you find and install packages as well as their dependencies. Though conda is written in Python, it can handle software written in any language. It runs on Windows, Linux and macOS and various platforms.
```

Conda isn't OS-agnostic, and there is no need to undersell it as being created for Python programs 10 years ago, e.g. the number of non-Python packages is not a niche.

Reply:
TIL. What does the "various platforms" bit mean in your suggestion, @jezdez? I'm also a big fan of the serial comma, so would suggest adding one between "Linux" and "macOS". "and various platforms" is just kind of hanging out at the end of that sentence, though, and I was unsure at first if it was related to macOS or if it was part of the OS list. Maybe we could do "It runs on Windows, Linux, and macOS, as well as various other platforms, such as X."

Reply:
Yeah, "various platforms" relates to the different CPU architectures and sub-architecturs conda is known to run on, like IBM's Power PC chips, the usual x86 and x86_64 (same but 64 bit) processors, the different derivates of the ARM platform (e.g. AWS' Graviton) and [most recently](https://github.com/conda/conda/commit/2020fb65812bb6c8813a25021d22958b607ed186#diff-b088be378fc4d3c8240b5a24fd9d51409bb64226eb874be18a10eafadec79f78) 64-bit RISC-V.

Reply:
Gotcha. I can see why you simplified it down. :) Maybe we could do "It runs on Windows, Linux, and macOS, as well as various platforms and architectures." and leave it at that.

Reply:
Sounds good to me!